prow_config.yaml

```diff
@@ -7,7 +7,8 @@ workflows:
     job_types:
       - presubmit
     include_dirs:
-    - pkg/api/operators/*
+    - pkg/api/operators/apis/studyjob
```

Comment:
Maybe pkg/api/operators/apis/studyjob/* is better

Reply:
done! thanks

Reply:
Yeah, it does a regex match, so this needs to have the /*

Reply:
See https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/run_e2e_workflow.py#L187